src/index.tsx

```diff
@@ -5,6 +5,7 @@ import Loadable from 'react-loadable';
 import createApp from './app';
 import { assertWindowDefined } from './app/utils';
 import config from './config';
+import './content.css';
```

Comment:
so we're unconditionally loading the generic styles, but then conditionally loading specific styles on top of that? and presumably the generic styles would then target a selector that didn't exist?

Reply:
yes, the selector in `content.css` is `#main-content[data-is-generic="true"]` so the CSS is not applied when the book is not configured to use the generic style.